app/controllers/post_comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class PostCommentsController < ApplicationController
+  before_action :logged_in_user
+  before_action :load_commentable, only: :create
+
+  def create
+    @post_comments = @commentable.post_comments.new post_comment_params
```

Comment:
```suggestion
    @post_comments = @commentable.post_comments.new post_comment_params.merge user_id: current_user.id
```
vs đặt lại tên biến nhé
số nhiều hay số ít

Reply:
vang chi